core/src/block_sync.rs

```diff
@@ -181,11 +184,34 @@ pub mod message {
                         return;
                     }
 
-                    let mut blocks = block_sync.sumeragi.blocks_after_hash(*hash);
-                    blocks.truncate(block_sync.block_batch_size as usize);
+                    let mut h = 1;
```

Comment:
This is very much unreadable. First of all, `h` is completely opaque. Secondly, unless the person reading this sees the `println` statement, they wouldn't know that `h` stands for `height`.